src/shared-components/dropdown/index.tsx

```diff
@@ -37,10 +55,10 @@ class Dropdown extends React.Component {
     this.setState({ isOpen: !isOpen });
   };
 
-  onSelectChange = event => {
+  onSelectChange = (event: React.ChangeEvent<HTMLSelectElement>) => {
     const { onChange } = this.props;
-
-    const { value, selectedOptions } = event.target;
+    const {target} = event;
```

Comment:
Did this not auto-format to be `{ target }`? I've noticed this before in this repo... wonder if something is up

Reply:
It did not - and apparently I've gotten very dependent on our linting to fix my bad whitespace habits 😅